From acc799798362dae3bb8eac694041663f4073af12 Mon Sep 17 00:00:00 2001
From: kenshen112 <finalfantasytimothy@gmail.com>
Date: Sun, 22 May 2022 16:44:51 -0700
Subject: [PATCH] QtHost.cpp: Fixed Resources dir

Fixed location of Resources Dir to proper usr/share location
---
 .gitignore          | 1 +
 pcsx2-qt/QtHost.cpp | 8 +++++---
 2 files changed, 6 insertions(+), 3 deletions(-)

diff --git a/.gitignore b/.gitignore
index 9cb112d2367a..67d4a5ee6831 100644
--- a/.gitignore
+++ b/.gitignore
@@ -74,6 +74,7 @@ oprofile_data/
 /bin/**/*.lib
 /bin/**/*.pdb
 /bin/pcsx2
+/bin/pcsx2-qt
 /bin/PCSX2-linux.sh
 /bin/*ReplayLoader
 /bin/GS*.txt
diff --git a/pcsx2-qt/QtHost.cpp b/pcsx2-qt/QtHost.cpp
index c40c4dabcad5..c1328bc02fe5 100644
--- a/pcsx2-qt/QtHost.cpp
+++ b/pcsx2-qt/QtHost.cpp
@@ -130,12 +130,14 @@ void QtHost::SetAppRoot()
 
 void QtHost::SetResourcesDirectory()
 {
-#ifndef __APPLE__
+#ifdef __APPLE__
+	// On macOS, this is in the bundle resources directory.
+	EmuFolders::Resources = Path::Canonicalize(Path::Combine(EmuFolders::AppRoot, "../Resources"));
+#elif !defined(PCSX2_APP_DATADIR)
 	// On Windows/Linux, these are in the binary directory.
 	EmuFolders::Resources = Path::Combine(EmuFolders::AppRoot, "resources");
 #else
-	// On macOS, this is in the bundle resources directory.
-	EmuFolders::Resources = Path::Canonicalize(Path::Combine(EmuFolders::AppRoot, "../Resources"));
+	EmuFolders::Resources = Path::Canonicalize(Path::Combine(EmuFolders::AppRoot, PCSX2_APP_DATADIR "/resources"));
 #endif
 }
 
